src/game_battlealgorithm.cpp

```diff
@@ -1215,7 +1216,16 @@ bool Game_BattleAlgorithm::Skill::Execute() {
 		skill.scope == RPG::Skill::Scope_party ||
 		skill.scope == RPG::Skill::Scope_self;
 
-	this->revived = this->healing
+	bool heals_states = this->healing;
```

Comment:
At some point I'd like to get rid of the `healing` and `positive` concepts here. Battle actions should be more general and just have effects (+-stats, inflict/heal states, etc..)

Reply:
Healing and positive really don't work. Won't touch this here, too easy to break, but this needs a refactor